editors/code/package.json

```
@@ -0,0 +1,61 @@
+{
+  "name": "code",
+  "displayName": "code",
+  "description": "",
+  "version": "0.0.1",
+  "engines": {
+    "vscode": "^1.89.1"
+  },
+  "categories": [
+    "Other"
+  ],
+  "activationEvents": [
+    "onLanguage:sql"
+  ],
+  "main": "./dist/native.js",
+  "browser": "./dist/browser.js",
+  "contributes": {
+    "commands": [
+      {
+        "command": "code.helloWorld",
+        "title": "Hello World"
+      }
+    ]
+  },
```

nit: This command can be removed. 

```suggestion
  "contributes": {
    "commands": [
      {
        "command": "code.helloWorld",
        "title": "Hello World"
      }
    ]
  },
```